src/components/Account/Login/Login.component.js

```diff
@@ -49,6 +59,8 @@ export class Login extends Component {
 
     const isButtonDisabled = isLogging || !!loginStatus.success;
 
+    var showPassword = false;
```

Comment:
A state should be used for this. This is the reason why you need to type a new character to see the password.